script/libtest.bash

```diff
@@ -455,8 +456,13 @@ create_http_trigger(){
     if [ -n "$gateway" ]; then
         command="$command --gateway ${gateway}"
     fi
-    eval $command
+    if [ -n "$cors" ]; then
+        make -sC examples create-cors-trigger
```

Comment:
can you add a command here to explain why this behavior is different? Something like:
```
# TODO: use --cors-enable flag when there is a new version of Kubeless CLI
```